ietf/standard/templates/standard/standard_index_page.html

```diff
@@ -1,62 +1,78 @@
-      </div>
-    {% endif %}
-
-    {% if self.key_info %}
-      <div class="read-more-content">
-        {% include_block self.key_info %}
-      </div>
-    {% endif %}
+  {% if self.key_info or self.in_depth %}
+    <div class="no-js-hide u-max-text-width">
+      {% has_tabs self.key_info self.in_depth as show_tabs %}
+        {% if show_tabs %}
+          <ul class="nav nav-tabs text-uppercase text-center" id="tabs" role="tablist" aria-owns="key-info-tab in-depth-tab">
+            <li class="nav-item flex-grow-1" role="presentation">
+              <a class="nav-link active" id="key-info-tab" data-toggle="tab" href="#key-info" role="tab"
+                aria-controls="key-info" aria-selected="true">Key Info</a>
+            </li>
+            <li class="nav-item flex-grow-1" role="presentation">
+              <a class="nav-link" id="in-depth-tab" data-toggle="tab" href="#in-depth" role="tab" aria-controls="in-depth"
+                aria-selected="false">In Depth</a>
+            </li>
+          </ul>
+        {% endif %}
 
-    {% if self.in_depth %}
-      <div class="read-more-content-two">
-        {% include_block self.in_depth %}
+        <div class="mb-3">
+          <div class="{% if show_tabs %}mb-3 tab-content bg-white border border-top-0{% endif %}">
+            {% if self.key_info %}
+            <div id="key-info" class="{% if show_tabs %} tab-pane fade show active p-3" tabindex="0"
+              role="tabpanel" aria-labelledby="key-info-tab{% endif %}">
+              {{ self.key_info|safe }}
+            </div>
+            {% endif %}
+        
+            {% if self.in_depth %}
+            <div id="in-depth" class="{% if show_tabs %} tab-pane fade p-3" role="tabpanel"
+              aria-labelledby="in-depth-tab{% endif %}">
+              {{ self.in_depth|safe }}
+            </div>
+            {% endif %}
+          </div>
+        </div>
       </div>
     {% endif %}
-  </div>
 
 {% endblock head_content %}
 
 {% block content %}
 
-  <ul class="text-listing two-col">
+<div class="container">
+  <ul class="row list-unstyled mt-5">
     {% for child in self.children %}
-        <li>
-        <div class="content-text">
-          <h2 class="xlarge"><a href="{% pageurl child %}">{{ child.title }}</a></h2>
-          <p>{% if child.search_description %}{{ child.search_description }}{% else %}{{ child.introduction }}{% endif %}</p>
-        </div>
-        {% if child.feed_related_links.exists %}
-          <span class="feedback link-toggle">
-            <a class="see">Related</a>
-            <a class="hide">Close</a>
-          </span>
-          <ul>
-            <li class="content">
-              {% for related in child.feed_related_links.all %}
-                <li><a href="{{ related.link }}">{{ related.title }}</a></li>
-              {% endfor %}
-            </li>
-          </ul>
-        {% endif %}
-    </li>
+        <li class="col-12 col-lg-6 mb-4">
+          <div class="content-text">
```

Comment:
I think we should remove classes that aren't used

Reply:
Good point